ui/src/mocks/handlers.ts

```diff
@@ -25,8 +25,15 @@ const recipes: Recipe[] = [
 ];
 
 export const handlers = [
-  rest.get<Recipe>("/api/products?q=mil", (req, res, ctx) => {
-    return res(ctx.json([{ name: "milk" }] as Product[]));
+  rest.get<Recipe>("/api/products", (req, res, ctx) => {
+    const query = req.url.searchParams;
+    const q = query.get("q");
+
+    if (q == "mil") {
```

Comment:
what's this? looks like mock

Reply:
Yes, this is a mock. I'm using in tests.

Reply:
And we want this in repo? xD